docs/quick-tutorials/aws-eks-cni-mini.mdx

```diff
@@ -0,0 +1,161 @@
+---
+sidebar_position: 8
+title: AWS EKS CNI Mini-tutorial
+---
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Running the AWS VPC CNI plugin on EKS with Otterize. This mini-tutorial shows how to run the AWS VPC CNI plugin on EKS with Otterize.
+
+## Prerequisites
+
+* An EKS cluster with the AWS VPC CNI plugin installed. See [Installing the AWS VPC CNI plugin](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) for more information.
```

Comment:
This is the default really, so the key part here is to have network policies enabled.

Reply:
Do I need to change anything?